### DISCUSSION TITLE
Change the menu layout to account fopr poeple using screen in portrait mode with long screen

### DISCUSSION THREAD

**Discussion by DMTarmey** [TransformerOptimus/SuperAGI#741] [Ideas]
IM using my screen length wise to get all the code on the same page and in this configuration the agent and tool kit button are waiting my width of text, can those buttons be moved, also the right box on right can that also have a hide menu option, cheer for you consideration.

When in this mode there seem to be to many menus many of them could stack onto the left or right in 1 row reducing the screen space that is taken up.

**Comment by joshknopp** [comment]
Same principle for mobile browser. I'd like to monitor and tinker with agents on my phone via local network, but the layout is clearly intended for desktop landscape mode. Happy to engage and put a few cycles toward bootstrap-ifying for narrow screens. Is this worth opening an issue?
